CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.2.6] - 2023-05-29
+
+### Changed
+
+- Accepts either group_id or project_id for the sdk, batch or job. This is a patch preparing the depreciation of group_id.
```

Comment:
The changelog is intended for the user, I would phrase it differently, something like:

`"Groups" have been renamed as "Projects", hence URL endpoints and attribute names have been changed accordingly. For example the `group_id` of a batch has been renamed to `project_id`. Note that for backwards compatibility, the `group_id` is still exposed by the APIs as a duplicate of the `project_id`. 

However it is considered deprecated and will be removed in future releases.